### DISCUSSION TITLE
Lattice vectors from box size

### DISCUSSION THREAD

**Discussion by KP-303** [mir-group/allegro#90] [Q&A] [answered]
If I have a cubic box in my simulation with side length L and the origin is at the centre of the box, what are the lattice vectors that should go in the .extxyz file, is it just (L,0,0) (0,L,0) (0,0,L)? Sorry if this is a basic question, in the past I've only done simulations in the gas phase and I wasn't fully confident with what I found on google.

Thanks.

**Comment by terryfrankcombe** [comment, marked as answer]
Correct
